src/saveload/marker.rs

```diff
@@ -0,0 +1,209 @@
+//! Provides with `Marker` and `MarkerAllocator` traits
+
+use std::collections::HashMap;
+use std::hash::Hash;
+
+use shred::Resource;
+use {Component, DenseVecStorage, Entities, Entity, Join, ReadStorage, WriteStorage};
+
+use super::SerializableComponent;
+
+
+/// This trait should be implemetened by component which is gonna be used as marker.
+/// This marker should be set to entity that should be serialized.
+/// If serialization strategy needs to set marker to some entity it should use
+/// new marker allocated for `Marker::Allocator`.
+///
+/// # Example
+///
+/// ```rust,no_run
```

Comment:
Why `no_run`?

Reply:
Nothing to run here. No need to waste cpu

Reply:
Nothing to run here. It only needs to compile

Reply:
Please still run it.